lnwallet/size.go

```diff
@@ -0,0 +1,107 @@
+package lnwallet
+
+import (
+	"github.com/roasbeef/btcd/blockchain"
+)
+
+const (
+	WitnessFactor      = blockchain.WitnessScaleFactor
+	MaxTransactionCost = blockchain.MaxBlockCost / 10
+
+	// The weight(cost), which is different from the !size! (see BIP-141),
+	// is calculated as:
+	// Weight = 4 * BaseSize + WitnessSize (weight).
```

Comment:
More precisely, the weight is calculated as: `(strippedSize * 3 )+ totalSize`, where `strippedSize` is the serialized size of the transaction with all the witness data stripped out.